iotdb-client/client-py/iotdb/Session.py

```diff
@@ -1058,6 +1076,35 @@ def insert_aligned_tablets(self, tablet_lst):
                         self.connection_error_msg()
                     ) from None
 
+    def insert_relational_tablet(self, tablet):
+        """
+        insert one tablet, in a tablet, for each timestamp, the number of measurements is same
+            for example three records in the same device can form a tablet:
+                timestamps,     m1,    m2,     m3
+                         1,  125.3,  True,  text1
+                         2,  111.6, False,  text2
+                         3,  688.6,  True,  text3
+        :param tablet: a tablet specified above
+        """
+        request = self.gen_insert_relational_tablet_req(tablet)
+        try:
+            connection = self.get_connection(tablet.get_device_id())
```

Comment:
Better to add some comments to explain that it is the table name here actually, or change the method name to a more generic one.

Reply:
fixed